code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -707,6 +707,13 @@
 	target.mech_melee_attack(src, user)
 	TIMER_COOLDOWN_START(src, COOLDOWN_MECHA_MELEE_ATTACK, melee_cooldown)
 
+
+/// Driver alt clicks a turf, cancel lootpanel alt click
+/obj/vehicle/sealed/mecha/proc/on_click_alt(mob/user, atom/target, params)
+	SIGNAL_HANDLER
+
+	return COMSIG_MOB_CANCEL_CLICKON
```

Comment:
Doesn't this mean they can't alt click the mech to strafe at all now?

Maybe we should have a flag components can return like `CLICK_NO_LOOT_PANEL` which does default handling + don't open the loot panel if nothing reaches there?

Reply:
no, "alt click" in the original issue does not refer to clicking the mech, its just holding the button and moving

they could not alt-click their own mech while in the mech previously because it would fail the interaction check

Reply:
So what does `mecha/click_alt` do now?

Reply:
`/mecha/click alt` is referring to a player clicking alt on the mech from outside, actually, this is 100% prior behavior,,,,

Reply:
Why does it do `if(!(user in occupants))` in occupants? Am I confusing something? 😵‍💫

Reply:
for reference:
https://github.com/tgstation/tgstation/blob/9dd8b5b5abaf43a597e0f527aa9ee381fb78bc8c/code/modules/vehicles/mecha/mecha_movement.dm#L127-L131

Reply:
> Why does it do `if(!(user in occupants))` then? Am I confusing something? 😵‍💫

it's not you Melbs it's the code, the code is confused